compose/foundation/foundation/src/uikitMain/kotlin/androidx/compose/foundation/cupertino/CupertinoOverscrollEffect.kt

```diff
@@ -148,8 +151,8 @@ class CupertinoOverscrollEffect(
 
     private fun NestedScrollSource.toCupertinoScrollSource(): CupertinoScrollSource? =
         when (this) {
-            NestedScrollSource.Drag -> CupertinoScrollSource.DRAG
-            NestedScrollSource.Fling -> CupertinoScrollSource.FLING
```

Comment:
Why was it renamed? I might be slightly out of context here, could you please explain?

Reply:
Sure, the old values where deprecated and no longer supported.